### PR TITLE
ENG-1324 show candy type

### PR DESCRIPTION
**Pull request by aletex1994**
• Added Property type column

### CODE REVIEW

File: packages/candy_editor/src/assets/formTypes/IntForms/integersForm.tsx [hunk 1]
```diff
@@ -163,31 +181,41 @@ export const IntegersForm = (editor: CandyClassEditor) => {
           {editor.property.immutable ? (
             <>
               <Grid column={1}>
```
Comment (Motokoder) [inline packages/candy_editor/src/assets/formTypes/IntForms/integersForm.tsx:183]: Grid column 1 is identical in both cases, so it should be moved above line 181.
Screenshot 2023-02-14 at 5 02 16 PM

**Review by Motokoder — CHANGES_REQUESTED** [review]
In general, take a look at all of your ternary expressions. If there is identity code, mode it out. If they can be combined by adding smaller ternary expressions in the attributes, combine them.

File: packages/candy_editor/src/assets/formTypes/BoolForm/boolForm.tsx [hunk 2]
```diff
@@ -107,9 +107,14 @@ export const BoolForm = (editor: CandyClassEditor) => {
           {editor.property.immutable ? (
             <>
               <Grid column={1}>
-                <TextInput value={name} disabled={true} />
+                <span style={{ marginTop: 'auto', marginBottom: 'auto' }}>
```
[the remark this answers is not in the document]
Comment (aletex1994) [reply packages/candy_editor/src/assets/formTypes/BoolForm/boolForm.tsx:110]: MOved grid 1 above in all components

File: packages/candy_editor/src/assets/formTypes/IntForms/integersForm.tsx (hunk 1 above)
Comment (aletex1994) [reply packages/candy_editor/src/assets/formTypes/IntForms/integersForm.tsx:183]: moved above in all components.

Comment (aletex1994) [reply packages/candy_editor/src/assets/formTypes/IntForms/integersForm.tsx]: moved grid columns 1 & 2 above the ternary operator in all the components

**Review by Motokoder — APPROVED** [review]

**Pull request merged**